pkg/images/images.go

```diff
@@ -4,6 +4,7 @@ const (
 	DefaultImageRegistry       string = "quay.io/skupper"
 	RouterImageName            string = "skupper-router:main"
 	ServiceControllerImageName string = "service-controller:master"
+	ControllerPodmanImageName  string = "controller-podman:master"
```

Comment:
should we add it into the version command output?

Reply:
I take that back, it is probably better to add it when the update command implementation for podman is ready.